src/_P128_NeoPixelBusFX.ino

```diff
@@ -63,9 +63,9 @@
 
    nfx theatre color [backgroundcolor] [count] [speed]
 
-   nfx scan color [backgroundcolor] [speed]
+   nfx scan color [backgroundcolor] [speed] [startpixel] [endpixel]
 
-   nfx dualscan color [backgroundcolor] [speed]
+   nfx dualscan color [backgroundcolor] [speed] [startpixel] [endpixel]
 
```

Comment:
Thanks, looking good. Haven't tested yet, but will do that soon.

Can you also update the documentation in `docs/source/Plugin/P128_commands.repl`?

Reply:
Yes! I didn't see this file... I'm not very good with the github tonhuisman, sorry... I make another commit I think...

I build all and tested in ESP8266 and works fine.
Thank you for your rapid reply!